examples/source/1.components/amp-experiment.html

```diff
@@ -106,11 +106,10 @@
 <!-- ## Development tip -->
 <!--
   You can force an experiment to be in a specific variant by adding `amp-x-experiment-name` to the AMP page URL.
-  <ul>
-    <li><a href="<% hosts.platform %>/components/amp-experiment#amp-x-button-color-experiment=0">#amp-x-button-color-experiment=0</a></li>
-    <li><a href="<% hosts.platform %>/components/amp-experiment#amp-x-button-color-experiment=1">#amp-x-button-color-experiment=1</a></li>
-    <li><a href="<% hosts.platform %>/components/amp-experiment#amp-x-button-color-experiment=2">#amp-x-button-color-experiment=2</a></li>
-  </ul>
+
+  - [#amp-x-button-color-experiment=0](<% hosts.platform %>{{ doc.url.path }}?0#amp-x-button-color-experiment=0)
```

Comment:
Does `target=_blank work` which I'd prefer?

Reply:
See the comment above: `target="_blank"` doesn't seem to open a new tab/window if it's the same location with a fragment.

Reply:
🤦‍♂️ sorry for missing your reply. Let's remove the get parameter in that case and add a comment to manually reload the page.

Reply:
All good, just added a hint!